### PR TITLE
Fix apiservice reapply infinitely issue

### PR DESCRIPTION
**Pull request by xuezhaojun**
Signed-off-by: xuezhaojun <zxue@redhat.com>

Relevant issue: https://github.com/open-cluster-management-io/registration-operator/issues/157

### CODE REVIEW

File: pkg/helpers/helpers.go [hunk 1]
```diff
@@ -334,10 +335,16 @@ func ApplyDirectly(
 			result.Result, result.Changed, result.Error = ApplyMutatingWebhookConfiguration(
 				client.AdmissionregistrationV1(), t)
 		case *apiregistrationv1.APIService:
+			t.ObjectMeta.Annotations = make(map[string]string)
+			checksum := md5.Sum(t.Spec.CABundle)
+			t.ObjectMeta.Annotations["caBundle-checksum"] = string(checksum[:]) // to trigger the update when caBundle changed
```
Comment (qiujian16) [inline pkg/helpers/helpers.go:340]: I think we commonly use sha256 like this

```
fmt.Sprintf("%x", sha256.Sum256(data))
```

Comment (xuezhaojun) [reply pkg/helpers/helpers.go:340]: Yes, changed to use sha56.

**Pull request merged**